### ISSUE TITLE
cannot import name 'HTMLString'

### ISSUE THREAD

**Issue by Harpsichord1207** [dpgaspar/Flask-AppBuilder#1356]
### Environment

Flask-Appbuilder version: ` Flask-AppBuilder==2.3.2`

pip freeze output:
```
alembic==1.4.2
apache-airflow==1.10.10
apispec==1.3.3
argcomplete==1.11.1
attrs==19.3.0
Babel==2.8.0
bcrypt==3.1.7
boto3==1.12.43
botocore==1.15.43
cached-property==1.5.1
cattrs==0.9.2
certifi==2020.4.5.1
cffi==1.14.0
chardet==3.0.4
click==7.1.1
colorama==0.4.3
colorlog==4.0.2
configparser==3.5.3
croniter==0.3.31
cryptography==2.9.1
defusedxml==0.6.0
dill==0.3.1.1
docutils==0.15
Flask==1.1.2
Flask-Admin==1.5.4
Flask-AppBuilder==2.3.2
Flask-Babel==1.0.0
Flask-Caching==1.3.3
Flask-JWT-Extended==3.24.1
Flask-Login==0.4.1
Flask-OpenID==1.2.5
Flask-SQLAlchemy==2.4.1
flask-swagger==0.2.13
Flask-WTF==0.14.3
funcsigs==1.0.2
future==0.18.2
graphviz==0.13.2
gunicorn==19.10.0
idna==2.9
importlib-metadata==1.6.0
iso8601==0.1.12
itsdangerous==1.1.0
Jinja2==2.10.3
jmespath==0.9.5
json-merge-patch==0.2
jsonschema==3.2.0
lazy-object-proxy==1.4.3
lockfile==0.12.2
Mako==1.1.2
Markdown==2.6.11
MarkupSafe==1.1.1
marshmallow==2.21.0
marshmallow-enum==1.5.1
marshmallow-sqlalchemy==0.22.3
numpy==1.18.3
pandas==0.25.3
paramiko==2.7.1
pendulum==1.4.4
prison==0.1.3
psutil==5.7.0
pycparser==2.20
Pygments==2.6.1
PyJWT==1.7.1
PyNaCl==1.3.0
pyrsistent==0.16.0
pysftp==0.2.9
python-daemon==2.1.2
python-dateutil==2.8.1
python-editor==1.0.4
python3-openid==3.1.0
pytz==2019.3
pytzdata==2019.3
PyYAML==5.3.1
requests==2.23.0
s3transfer==0.3.3
setproctitle==1.1.10
six==1.14.0
SQLAlchemy==1.3.16
SQLAlchemy-JSONField==0.9.0
SQLAlchemy-Utils==0.36.3
sshtunnel==0.1.5
tabulate==0.8.7
tenacity==4.12.0
termcolor==1.1.0
text-unidecode==1.2
thrift==0.13.0
typing==3.7.4.1
typing-extensions==3.7.4.2
tzlocal==1.5.1
unicodecsv==0.14.1
urllib3==1.25.9
Werkzeug==0.16.1
WTForms==2.3.0
zipp==3.1.0
zope.deprecation==4.4.0
```

### Question Describe

I installed airflow, and run `airflow initdb`, got the following error
```
Traceback (most recent call last):
  File "/root/MyAirlfow/venv/bin/airflow", line 26, in <module>
    from airflow.bin.cli import CLIFactory
  File "/root/MyAirlfow/venv/lib64/python3.6/site-packages/airflow/bin/cli.py", line 71, in <module>
    from airflow.www_rbac.app import cached_app as cached_app_rbac
  File "/root/MyAirlfow/venv/lib64/python3.6/site-packages/airflow/www_rbac/app.py", line 28, in <module>
    from flask_appbuilder import AppBuilder, SQLA
  File "/root/MyAirlfow/venv/lib64/python3.6/site-packages/flask_appbuilder/__init__.py", line 6, in <module>
    from .base import AppBuilder  # noqa: F401
  File "/root/MyAirlfow/venv/lib64/python3.6/site-packages/flask_appbuilder/base.py", line 8, in <module>
    from .api.manager import OpenApiManager
  File "/root/MyAirlfow/venv/lib64/python3.6/site-packages/flask_appbuilder/api/manager.py", line 7, in <module>
    from flask_appbuilder.baseviews import BaseView
  File "/root/MyAirlfow/venv/lib64/python3.6/site-packages/flask_appbuilder/baseviews.py", line 21, in <module>
    from .forms import GeneralModelConverter
  File "/root/MyAirlfow/venv/lib64/python3.6/site-packages/flask_appbuilder/forms.py", line 17, in <module>
    from .fieldwidgets import (
  File "/root/MyAirlfow/venv/lib64/python3.6/site-packages/flask_appbuilder/fieldwidgets.py", line 3, in <module>
    from wtforms.widgets import html_params, HTMLString
ImportError: cannot import name 'HTMLString'
```
it shows that `flask_appbuilder` tried to import `HTMLString` from `wtforms.widgets` and failed.

**Comment by JulesDe** [comment]
In the latest wtforms version `2.3.0` (it was just released yesterday), `HTMLString` was removed and replaced with `markupsafe.Markup`: https://github.com/wtforms/wtforms/pull/400

If you use `WTForms==2.2.1` it should work fine for now.

**Comment by dpgaspar** [comment]
Hi @Harpsichord1207,

Thanks for the heads up, looking at Airflow the pinned version there is correct: https://github.com/apache/airflow/blob/master/requirements/requirements-python3.6.txt#L32

Better to use the pinned requirements

Yet this should be addressed here

**Comment by dpgaspar** [comment]
This seems to have been fixed on 2.3.1: https://github.com/wtforms/wtforms/blob/master/CHANGES.rst#version-231
